spec/fake_app.rb

```diff
@@ -2,16 +2,12 @@
 module PaulRevere
   class Application < Rails::Application
     config.secret_key_base = 'test'
+    config.paths['config/database'] = ['spec/support/database.yml']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.